docs/autolinking.md

```diff
@@ -1,20 +1,14 @@
 # Autolinking
 
-Autolinking is a mechanism built into CLI that allows adding a dependency with native components for React Native to be as simple as:
+React Native libraries usually come with platform-specific (native) code. Autolinking is a mechanism that allows your project to discover and use this code.
 
-```sh
-yarn add react-native-webview
-```
+Add a library using your favorite package manager and run the build. That's it. No more editing build config files to use native code.
 
-> Autolinking is a replacement for [`react-native link`](./linking.md) that brings new features (such as ability to easily integrate native dependencies on iOS) and fixes some of the long-standing issues.
+> Autolinking is a replacement for [`react-native link`](./linking.md). You're expected to `unlink` native dependencies (if any) for it to work well. It will also ease React Native upgrades in the future.
```

Comment:
"If you have been using React Native before version 0.60, please `unlink` native dependencies if you have any from a previous install. <Example>".